spec/spec2_spec.cr

```diff
@@ -1,521 +0,0 @@
-require "./spec_helper"
```

Comment:
This file is fully re-written in a unit-test fashion and sits in `./unit/simple_dsl_spec.cr`